repository/git.go

```diff
@@ -985,3 +985,37 @@ func (repo *GitRepo) FetchAndReturnNewReviewHashes(remote, notesRefPattern,
 	}
 	return updatedReviews, nil
 }
+
+func (repo *GitRepo) GetColorBool(name string) bool {
+	ok := repo.runGitCommandInline("config", "--get-colorbool", name)
+	return (ok == nil)
+}
+
+func (repo *GitRepo) GetColor(name, default_value string) string {
+	var res string
+	var ok error
+	if default_value == "" {
+		res, ok = repo.runGitCommand(
```

Comment:
In this case I would recommend building the command args first, appending to them if the default is provided, and then only having one line that invokes the command.

e.g.

```go
	args := []string{"config", "--type=color", "-z"}
	if defaultValue != "" {
		args = append(args, "--default", defaultValue)
	}
	args = append(args, "--get", name)
	res, err := repo.runGitCommand(args...)
```